android/src/main/java/io/wifi/p2p/WiFiP2PManagerModule.java

```diff
@@ -90,6 +90,7 @@ public void onGroupInfoAvailable(WifiP2pGroup group) {
   @ReactMethod
   public void init(Promise promise) {
     if (manager != null) { // prevent reinitialization
+      promise.reject("You should only call initialize once.");
```

Comment:
Such signature is not deprecated? I thought it requires code + error message 🤔

Reply:
<img width="762" alt="image" src="https://github.com/kirillzyusko/react-native-wifi-p2p/assets/22820318/8b5d5b46-956b-4ca6-a4f4-3ed8b2ce95b3">

I think you can use `0x2` code 🙂